[TEST-FAILURE] QueueAdvancedTest

com.hazelcast.collection.impl.queue.QueueAdvancedTest.testPutInterruption
```
Stacktrace

java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.hazelcast.collection.impl.queue.QueueAdvancedTest.testPutInterruption(QueueAdvancedTest.java:629)
```
https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast/685/testReport/junit/com.hazelcast.collection.impl.queue/QueueAdvancedTest/testPutInterruption/

com.hazelcast.collection.impl.queue.QueueAdvancedTest.testTakeInterruption
```
Stacktrace

java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.hazelcast.collection.impl.queue.QueueAdvancedTest.testTakeInterruption(QueueAdvancedTest.java:599)
```
https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast/685/testReport/junit/com.hazelcast.collection.impl.queue/QueueAdvancedTest/testTakeInterruption/

--- a/hazelcast/src/test/java/com/hazelcast/collection/impl/queue/QueueAdvancedTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/collection/impl/queue/QueueAdvancedTest.java
@@ -580,9 +580,11 @@ public class QueueAdvancedTest extends HazelcastTestSupport {
 
         final AtomicBoolean interrupted = new AtomicBoolean();
 
+        final CountDownLatch latch = new CountDownLatch(1);
         Thread t = new Thread() {
             public void run() {
                 try {
+                    latch.countDown();
                     queue.take();
                 } catch (InterruptedException e) {
                     interrupted.set(true);
@@ -591,7 +593,7 @@ public class QueueAdvancedTest extends HazelcastTestSupport {
         };
         t.start();
 
-        sleepSeconds(1);
+        latch.await(10, TimeUnit.SECONDS);
         t.interrupt();
         t.join(5000);
 
@@ -610,9 +612,11 @@ public class QueueAdvancedTest extends HazelcastTestSupport {
 
         assertTrue(queue.offer("item"));
 
+        final CountDownLatch latch = new CountDownLatch(1);
         Thread t = new Thread() {
             public void run() {
                 try {
+                    latch.countDown();
                     queue.put("item");
                 } catch (InterruptedException e) {
                     interrupted.set(true);
@@ -621,7 +625,7 @@ public class QueueAdvancedTest extends HazelcastTestSupport {
         };
         t.start();
 
-        sleepSeconds(1);
+        latch.await(10, TimeUnit.SECONDS);
         t.interrupt();
         t.join(5000);
 

